.github/workflows/publish-docker.yaml

```diff
@@ -20,6 +20,7 @@ on:
   push:
     branches:
       - main
+  workflow_dispatch:
```

Comment:
Don't we need a version parameter to manually build?

Reply:
> Don't we need a version parameter to manually build?

No. We would either rebuild the latest master branch or need code changes in latest master. Rebuilding an old commit doesn't make sense.

Reply:
I see. You want to rebuild the latest for local branch? Is that the purpose?

Reply:
yes

Reply:
So, this would not work for fork repo, mostly it is for committers' branch.